packages/dcos-oauth/extra/dcos_add_user.py

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
@jgehrcke Could you please update `README.md`? The instruction doesn't work anymore on master branch.

https://github.com/dcos/dcos/blob/master/README.md#instructions